edx_django_utils/private_utils.py

```diff
@@ -0,0 +1,57 @@
+"""
+Private utility methods.
+
+These utilities are designated for internal use only (i.e. inside
+edx-django-utils).  In some cases, these utilities could be elevated to public
+methods once they get some burn-in time and find a more permanent home in the
+library.
```

Comment:
Nit: I think it's fine to make a module for middleware utilities and have no public methods in it (just _ prefixed ones), to indicate internal use, instead of having a global catch bucket for internal utilities.

Reply:
I wanted to avoid ending up with a lot of directories each of which might have 1 helper function, but maybe it is fine.  If I set up a whole directory, do we want to make this public?  For example, the DRF Middleware I added could use this.

Reply:
Actually, let's just leave it. It's probably not worth the length of this conversation thread. :P